backend/src/main/resources/schema.sql

```diff
@@ -32,9 +32,17 @@ CREATE TABLE Members
     PRIMARY KEY (account, organization)
 );
 
+CREATE TABLE UserCustomers (
+    user VARCHAR(128),
```

Comment:
`user` is apparently a reserved keyword in postgresql (yeah...), so call it something else, maybe `email`?